spec/services/step_by_step_page_reverter_spec.rb

```diff
@@ -83,6 +83,12 @@
         expect(step_by_step_page.steps[1].position).to eq(2)
         expect(step_by_step_page.steps[2].position).to eq(3)
       end
+
+      describe "#contents" do
+        it "saves the contents of a paragraph" do
+          expect(step_by_step_page.steps[0].contents).to eq("A paragraph of text in the first step.")
```

Comment:
Can we test for the multiline thing?

Reply:
I have a test for a multi-line, multi-list, multi-bullets, with and without context monstrosity of a step in a later commit.

Reply:
@sihugh I've updated the test to store multiple paragraphs. I felt it was appropriate as I mentioned it in the comments. Cheers!